infrastructure/terraform/module-ecs-services/variables.tf

```diff
@@ -2,20 +2,28 @@ variable "stack_name" {}
 variable "name_prefix" {}
 variable "environment" {}
 
+variable "docker_registry" {}
+variable "app_version_test_data_generator" {}
```

Comment:
Can this just be `version`? We can glean context from the repository

Reply:
It could be. The only thing to consider is that it's set at the pipeline and passed through as a TF_VAR. So may need some extra context so it makes sense there too.

Reply:
Might be one other consideration here too. Billy mentioned that some stacks in future (which hopefully can use this as an example) may have more than one application (and hence version) in the stack. This variable had originally been more generic but was changed in this commit: https://github.com/companieshouse/test-data-generator-stack/pull/2/commits/a8f2a2dcbf6190810377155ac5d47f531df98166

Reply:
As per discussion, variable name has now been changed from 'app_version_test_data_generator' to 'release_version'.
('version' on its own is a reserved word)